packages/netlify-cms-backend-github/package.json

```diff
@@ -17,10 +17,17 @@
   "scripts": {
     "develop": "yarn build:esm --watch",
     "build": "cross-env NODE_ENV=production webpack",
-    "build:esm": "cross-env NODE_ENV=esm babel src --out-dir dist/esm --ignore **/__tests__ --root-mode upward"
+    "build:esm": "cross-env NODE_ENV=esm babel src --out-dir dist/esm --ignore **/__tests__ --root-mode upward",
+    "createFragmentTypes": "node scripts/createFragmentTypes.js"
```

Comment:
script to generate `fragmentTypes.js`. Apollo will use `fragmentTypes.js` to better support [union types](https://www.apollographql.com/docs/react/advanced/fragments/#fragments-on-unions-and-interfaces)